sunpy/net/helio/hec.py

```diff
@@ -223,7 +219,7 @@ def make_table_list(self):
         --------
         >>> from sunpy.net.helio import hec
         >>> hc = hec.HECClient()  # doctest: +REMOTE_DATA
-        >>> hc.make_table_list()  # doctest: +REMOTE_DATA
+        >>> hc.make_table_list()  # doctest: +REMOTE_DATA +SKIP
```

Comment:
But my skip?!

Reply:
this one is real. It's just nonsense.

Reply:
oh that's it, this function needs user input, so it can't be run as a doctest.